docker/compute_worker/watch.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+watchmedo shell-command --patterns="*.html" \
```

Comment:
How often will this update? Doing `watchmedo shell-command --help` I was able to see some "hidden flags" (not on their docs anywhere that I could see..):

```
❯ watchmedo shell-command --help
usage: watchmedo shell-command [-h] [-c COMMAND] [-p PATTERNS]
                               [-i IGNORE_PATTERNS] [-D] [-R]
                               [--interval TIMEOUT] [-w] [-W]
                               [--debug-force-polling]
                               [directories [directories ...]]

    Subcommand to execute shell commands in response to file system events.

    :param args:
        Command line argument options.
    

positional arguments:
  directories           directories to watch (default: '.')

optional arguments:
  -h, --help            show this help message and exit
  -c COMMAND, --command COMMAND
                        shell command executed in response to matching events.
                        These interpolation variables are available to your
                        command string:: ${watch_src_path} - event source
                        path; ${watch_dest_path} - event destination path (for
                        moved events); ${watch_event_type} - event type;
                        ${watch_object} - ``file`` or ``directory`` Note::
                        Please ensure you do not use double quotes (") to
                        quote your command string. That will force your shell
                        to interpolate before the command is processed by this
                        subcommand. Example option usage:: --command='echo
                        "${watch_src_path}"' (default: -)
  -p PATTERNS, --pattern PATTERNS, --patterns PATTERNS
                        matches event paths with these patterns (separated by
                        ;). (default: '*')
  -i IGNORE_PATTERNS, --ignore-pattern IGNORE_PATTERNS, --ignore-patterns IGNORE_PATTERNS
                        ignores event paths with these patterns (separated by
                        ;). (default: '')
  -D, --ignore-directories
                        ignores events for directories (default: False)
  -R, --recursive       monitors the directories recursively (default: False)
  --interval TIMEOUT, --timeout TIMEOUT
                        use this as the polling interval/blocking timeout
                        (default: 1.0)
  -w, --wait            wait for process to finish to avoid multiple
                        simultaneous instances (default: False)
  -W, --drop            Ignore events that occur while command is still being
                        executed to avoid multiple simultaneous instances
                        (default: False)
  --debug-force-polling
                        [debug] forces polling (default: False)
```

I think we want _at least_ `--wait` here so we don't do this multiple times at once, potentially having an old state override a new state. `--drop` would be kind of lame to miss a state change..

Reply:
There's an issue here for adding a `--debounce` .. maybe something we could do at some point if we're feeling crazy :) or if @gibsonbailey's brothers want an open source project

https://github.com/gorakhargosh/watchdog/issues/315